inst/templates/BS3/footer.html

```diff
@@ -1,7 +1,7 @@
 <div class="copyright">
-  <p>{{#package}}Developed by {{{authors}}}.{{/package}}</p>
+  <p>{{#footer}}{{#left}}{{{.}}}{{/left}}{{/footer}}</p>
```

Comment:
Move the `{{#footer}}` out to wrap around the entire block? We also should rename the div classes when we switch to BS4.